### PR TITLE
💄 Newsletter Signup - implement redesign

### LINKED ISSUE #755: 💄 Newsletter Signup - implement redesign
Reskin newsletter sign up component to look like

Image

Note: should be able to use Tina to drop component on other pages e.g. home page

### PR DESCRIPTION
**Pull request by KristenHu**
Closed #755 - Reskin the signup component

Affected routes:

-`/newsletters`
image

### CODE REVIEW

File: components/blocks/subNewsLetters.tsx [hunk 1]
```diff
@@ -105,9 +106,11 @@ export const SubNewsLetters = ({ headerText, subscribeButtonText }) => {
...
 
   return (
-    <div>
-      <p className="mb-2">{headerText}</p>
-      <div className="container sm:w-full sm:max-w-full md:w-1/4">
+    <div className="flex flex-col items-center justify-center">
+      <h2 className="mb-2 mt-6 font-helvetica text-4xl font-medium text-sswBlack" dangerouslySetInnerHTML={{ __html: headerText }}></h2>
+      <div className="mb-5 text-base text-sswBlack">{subscribeSubTitle}</div>
+      <div className="container flex justify-center sm:w-full sm:max-w-full md:w-3/4">
+        {/* <div className=""> */}
```
Comment (bradystroud) [inline components/blocks/subNewsLetters.tsx]: Can we remove this?

File: components/blocks/subNewsLetters.tsx [hunk 2]
```diff
@@ -120,15 +123,18 @@ export const SubNewsLetters = ({ headerText, subscribeButtonText }) => {
...
-        <div className="flex justify-center">
+        <span className="w-80">
+          {inputRender(
+            email,
+            placeholder.email,
+            Object.keys(placeholder)[2],
+            handleEmailChange
+          )}
+        </span>
+        {/* </div> */}
```
Comment (bradystroud) [inline components/blocks/subNewsLetters.tsx]: And this :)

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Review by bradystroud — CHANGES_REQUESTED** [review]
image

Make this icon look like the Submit button on the booking form

image

**Figure: ✅ New icon looks much nicer**

**Review by bradystroud — CHANGES_REQUESTED** [review]
Hey @KristenHu 

Love the new design 🙂

1. The signup button should be disabled while the signup is in progress (like the submit button on the booking form)
2. Getting an error when signing up 😕


**Figure: signup not working**

File: components/blocks/subNewsLetters.tsx (hunk 1 above)
Comment (KristenHu) [reply components/blocks/subNewsLetters.tsx]: Done, sorry about that😊

File: components/blocks/subNewsLetters.tsx (hunk 2 above)
Comment (KristenHu) [reply components/blocks/subNewsLetters.tsx]: Done

**Comment by KristenHu** [comment]
> Hey @KristenHu
> 
> Love the new design 🙂
> 
> 1. The signup button should be disabled while the signup is in progress (like the submit button on the booking form)
> 2. Getting an error when signing up 😕
> 
>  **Figure: signup not working**


1. The signup button should be disabled while the signup is in progress (like the submit button on the booking form)
**Done.**
2. Getting an error when signing up 😕
**Done** - Fixed with @amankumarrr  😃
image

**Review by bradystroud — APPROVED** [review]
LGTM 🥳

**Pull request merged**